parlai/tasks/coqa/build.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+# Download and build the data if it does not exist.
+
+import parlai.core.build_data as build_data
+import os
+import json
+
+tfname = 'coqa-train-v1.0.json'
+dfname = 'coqa-dev-v1.0.json'
+url = 'https://nlp.stanford.edu/data/coqa/'
+
+
+def make_parlai_format(outpath, dtype, data):
+    print('building parlai:' + dtype)
+    fout = open(os.path.join(outpath, dtype + '.txt'), 'w')
+    for each in data:
+        output = ""
+        story = each['story'].replace('\n', '\\n')
+        for question, ans in zip(each['questions'], each['answers']):
+            question_txt = story + '\\n' + question['input_text'] \
```

Comment:
don't use `\` to end lines. Either wrap in a parentheses or don't wrap lines.

In this case I recommend just spelling it out, because I'd prefer a comment
```python
if question['turn_id'] == 1:
    # include the context in the very first turn
    question_txt = story + '\\n' + question['input_text']
else:
    question_txt = question['input_text']
```